Description: use the correct type for m_lLen
 pdf_long happens to be compatible with long on 64 bit architectures, but
 clashes on 32 bits.  Given that m_lLen is only used in place where pdf_long
 should be used instead, let's change the definition.
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: https://sourceforge.net/p/podofo/mailman/podofo-users/thread/20170215164219.wjviqkdptn4uag6s%40mapreri.org/#msg35667897
Last-Update: 2017-02-15

--- a/test/unit/EncryptTest.h
+++ b/test/unit/EncryptTest.h
@@ -87,7 +87,7 @@
 
  private:
   char* m_pEncBuffer;
-  long  m_lLen;
+  PoDoFo::pdf_long m_lLen;
   int   m_protection;
   
 };
